forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -292,7 +340,24 @@ class FormResourceTaskVariablesbyApplicationId(Resource):
     @profiletime
     def get(application_id: int):
         """The method retrieves task variables based on application id."""
-        return (
-            ApplicationService.get_application_form_mapper_by_id(application_id),
-            HTTPStatus.OK,
-        )
+        try:
+            tenant_key = g.token_info.get("tenantKey")
```

Comment:
I created a decorator in forms-flow-ai-admin to use the user context. You could add it here too;
Reference : https://github.com/AOT-Technologies/forms-flow-ai-admin/blob/develop/admin_api/utils/user_context.py
How to use : https://github.com/AOT-Technologies/forms-flow-ai-admin/blob/develop/admin_api/services/tenant.py#L20

Nothing wrong with current code; just a suggestion.

Reply:
The decorator approach looks clean, will use it here. Thanks